.github/workflows/issue-sync.yml

```
@@ -0,0 +1,268 @@
+    name: Sync GitHub Issue → TODO.md
+    if: github.event_name == 'issues'
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    permissions:
+      contents: write
+      issues: read
+
+    steps:
+      - name: Check issue title format
+        id: check-issue
+        run: |
```

actionlint (1.7.10)

[error] "github.event.issue.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)